src/main/migrations/changesets/change_execution_unique_constraint.xml

```diff
@@ -5,8 +5,12 @@
 
     <changeSet author="tjeandet" id="change_execution_unique_constraint">
         <comment>
-            Change unique constraint for Execution Table to include INDEX column.
+            Change unique constraint for Execution Table to include INDEX column.  For MySQL this requires
+            first dropping the foreign key constraint, which we then restore after adding back the enhanced
+            unique constraint.  Both the unique and foreign key constraints are added back with shorter
+            Oracle-friendly (i.e. not more than 30 character) names.
         </comment>
+        <dropForeignKeyConstraint baseTableName="EXECUTION" constraintName="FK_EXECUTION_WORKFLOW_EXECUTION_ID" />
```

Comment:
I know changing existing changesets is usually a Bad Thing, but this is off on the scatter-gather branch and shouldn't have been seen by a non-transient DB so I rationalized it as okay.  :smile: